Mexican Spanish Translation

TheLegendofSaram opened the pull request:
Just a translation for the mod

TheLegendofSaram committed:
Mexican Spanish Translation

Just a translation for the mod

chronosacaria commented on common/src/main/resources/assets/simplyswords/lang/es_mx.json:
```diff
@@ -0,0 +1,1722 @@
+  "item.simplyswords.mythicmetals_compat.prometheum.prometheum_halberd": "Alabarda de prometio",
+
+
+  "item.simplyswords.mythicmetals_compat.quadrillum.quadrillum_claymore": "Mandoble de cuadrilio",
```
En cuanto a las líneas 289 a 303, «quadrillium» debería reconsiderarse como una traducción viable. La lógica es la siguiente:
1. El término original no es «quadrillium», sino «quadrillum». 
2. Como tal, «quadrilo» sería probablemente una mejor traducción que mantiene la estructura silábica lo suficientemente similar como para no confundir al usuario. 
3. Además, no hay ningún riesgo en utilizar «cuadrilo», ya que no existe esta palabra en los principales dialectos del español.

TheLegendofSaram replied:
Okkk, cambiare todos los terminos a quadrillium, tambien concuerdo que suena mejor y mantiene mejor la estructura

chronosacaria commented on common/src/main/resources/assets/simplyswords/lang/es_mx.json:
```diff
@@ -0,0 +1,1722 @@
+  "text.autoconfig.simplyswords_main.option.weapon_attributes.durasteel_damageModifier":                                "Modificador de daño base de duraacero",
+  "text.autoconfig.simplyswords_main.option.weapon_attributes.osmium_damageModifier":                                   "Modificador de daño base de osmio",
+  "text.autoconfig.simplyswords_main.option.weapon_attributes.prometheum_damageModifier":                               "Modificador de daño base de prometio",
+  "text.autoconfig.simplyswords_main.option.weapon_attributes.quadrillum_damageModifier":                               "Modificador de daño base de cuadrilio",
```
«quadrillium» debería reconsiderarse como una traducción viable. La lógica es la siguiente:
1. El término original no es «quadrillium», sino «quadrillum».
2. Como tal, «quadrilo» sería probablemente una mejor traducción que mantiene la estructura silábica lo suficientemente similar como para no confundir al usuario.
3. Además, no hay ningún riesgo en utilizar «cuadrilo», ya que no existe esta palabra en los principales dialectos del español.

TheLegendofSaram replied:
Yep, voy a cambiar todos los terminos de cuadrilo a "Quadrillium"

chronosacaria reviewed (changes requested):
Aparte de los dos puntos que se señalan para corrección, esta traducción es fantástica. Una vez resueltos los puntos mencionados, podré aprobar la traducción. Gracias por su contribución.

TheLegendofSaram committed:
es_mx update

TheLegendofSaram committed:
Mexican Spanish Translation for the runic grimoire

Tell me if you find any incosistency with the main translation.
This should also work with all of the other spanish variants, es_ar, es_es, es_ve, etc.

TheLegendofSaram commented:
Hey I also translated the grimoire, to es_mx.
This translations should also work for all of the other spanish variants es_es, es_ar, es_ve, es_cl, etc.

TheLegendofSaram commented:
And sorry for the late responce, i didnt see the email of the replay
